test/test_helper.rb

```diff
@@ -1,19 +1,54 @@
 # Configure Rails Environment
-ENV["RAILS_ENV"] = "test"
+ENV['RAILS_ENV'] = 'test'
 
-require File.expand_path("../dummy/config/environment.rb",  __FILE__)
-require "rails/test_help"
+require File.expand_path('../dummy/config/environment.rb',  __FILE__)
+require 'rails/test_help'
+
+require 'minitest/reporters'
+Minitest::Reporters.use!
 
 Rails.backtrace_cleaner.remove_silencers!
 
 # Load support files
-Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
+Dir['#{File.dirname(__FILE__)}/support/**/*.rb'].each { |f| require f }
 
 # Load fixtures from the engine
 if ActiveSupport::TestCase.method_defined?(:fixture_path=)
-  ActiveSupport::TestCase.fixture_path = File.expand_path("../fixtures", __FILE__)
+  ActiveSupport::TestCase.fixture_path = File.expand_path('../fixtures', __FILE__)
```

Comment:
I could put back the double quotes if that would be a better choice.  Not sure about `Style/ExpandPathArguments: Use expand_path('fixtures', dir) instead of expand_path('../fixtures', FILE).` and I didn't get a chance to research it yet.

Reply:
Hound's comment seem to be slighly off due to markdown support. What it's suggesting is:
`File.expand_path('fixtures', __dir__)`. Without those missing double underscores (`__`) it would probably error.
If it's not too much trouble, you can update. But otherwise, feel free to ignore. I have to upgrade Rubocop/Hound config a bit anyways.